File: RustRegistration/src/keygen.rs
use solana_sdk::{signature::{Keypair, Signer}, pubkey::Pubkey};

pub fn keygen() {
    // Create a new keypair
    let kp = Keypair::new();
    println!(
        "You've generated a new Solana wallet: {}",
        kp.pubkey().to_string()
    );
    println!("");
    println!("To save your wallet, copy and paste the following into a JSON file:");
    println!("{:?}", kp.to_bytes());
}

File: ts/bridge_to_turbin3_prereq/keygen.ts
import { Keypair } from "@solana/web3.js";

//Generate a new keypair
let kp = Keypair.generate()
console.log(`You've generated a new Solana wallet: ${kp.publicKey.toBase58()}`);
console.log(`To save your wallet, copy and paste the following Solana Wallet Secret Key into a JSON file: ${kp.secretKey}]`);

// terminal -> yarn keygen